[CRITICAL] KERNEL(6734,7f3b437e6700,python):2022-08-11-20:12:52.829.381 [mindspore/ccsrc/plugin/device/cpu/kernel/pad_cpu_kernel.cc:102] LaunchKernel] For 'Pad', output buffer memset failed.
[ERROR] DEVICE(6734,7f3b437e6700,python):2022-08-11-20:12:52.829.526 [mindspore/ccsrc/runtime/pynative/op_executor.cc:158] WorkerLoop] Run lazy task failed, error message:mindspore/ccsrc/plugin/device/cpu/kernel/pad_cpu_kernel.cc:102 LaunchKernel] For 'Pad', output buffer memset failed.
本轮突变失败
Traceback (most recent call last):
  File "/HNAS2022.7.24/main.py", line 50, in <module>
    torch_tf_max_diff,torch_tf_mean_diff,torch_mindspore_max_diff,torch_mindspore_mean_diff,tf_mindspore_max_diff,tf_mindspore_mean_diff,avg_max_diff,avg_mean_diff = worker.excute()
  File "/HNAS2022.7.24/DataStruct/worker.py", line 14, in excute
    torch_tf_max_diff,torch_tf_mean_diff,torch_mindspore_max_diff,torch_mindspore_mean_diff,tf_mindspore_max_diff,tf_mindspore_mean_diff,avg_max_diff,avg_mean_diff=calFitness(g)
  File "/HNAS2022.7.24/Method/calFitness.py", line 12, in calFitness
    avg_max_diff,avg_mean_diff=exe_module()
  File "/HNAS2022.7.24/Method/module_executor.py", line 125, in exe_module
    mindspore_output = mindspore_net(mindspore_input)
  File "/root/anaconda3/envs/icse2023/lib/python3.9/site-packages/mindspore/nn/cell.py", line 612, in __call__
    raise err
  File "/root/anaconda3/envs/icse2023/lib/python3.9/site-packages/mindspore/nn/cell.py", line 609, in __call__
    output = self._run_construct(cast_inputs, kwargs)
  File "/root/anaconda3/envs/icse2023/lib/python3.9/site-packages/mindspore/nn/cell.py", line 429, in _run_construct
    output = self.construct(*cast_inputs, **kwargs)
  File "/HNAS2022.7.24/Method/Models/testnet_mindspore.py", line 498, in construct
    out = self.cell_2(out)
  File "/root/anaconda3/envs/icse2023/lib/python3.9/site-packages/mindspore/nn/cell.py", line 612, in __call__
    raise err
  File "/root/anaconda3/envs/icse2023/lib/python3.9/site-packages/mindspore/nn/cell.py", line 609, in __call__
    output = self._run_construct(cast_inputs, kwargs)
  File "/root/anaconda3/envs/icse2023/lib/python3.9/site-packages/mindspore/nn/cell.py", line 429, in _run_construct
    output = self.construct(*cast_inputs, **kwargs)
  File "/HNAS2022.7.24/Method/Models/testnet_mindspore.py", line 344, in construct
    result = nn.layer.Sigmoid()(input)
  File "/root/anaconda3/envs/icse2023/lib/python3.9/site-packages/mindspore/nn/cell.py", line 612, in __call__
    raise err
  File "/root/anaconda3/envs/icse2023/lib/python3.9/site-packages/mindspore/nn/cell.py", line 609, in __call__
    output = self._run_construct(cast_inputs, kwargs)
  File "/root/anaconda3/envs/icse2023/lib/python3.9/site-packages/mindspore/nn/cell.py", line 429, in _run_construct
    output = self.construct(*cast_inputs, **kwargs)
  File "/root/anaconda3/envs/icse2023/lib/python3.9/site-packages/mindspore/nn/layer/activation.py", line 612, in construct
    return self.sigmoid(x)
  File "/root/anaconda3/envs/icse2023/lib/python3.9/site-packages/mindspore/ops/primitive.py", line 294, in __call__
    return _run_op(self, self.name, args)
  File "/root/anaconda3/envs/icse2023/lib/python3.9/site-packages/mindspore/common/api.py", line 90, in wrapper
    results = fn(*arg, **kwargs)
  File "/root/anaconda3/envs/icse2023/lib/python3.9/site-packages/mindspore/ops/primitive.py", line 754, in _run_op
    output = real_run_op(obj, op_name, args)
RuntimeError: mindspore/ccsrc/plugin/device/cpu/kernel/pad_cpu_kernel.cc:102 LaunchKernel] For 'Pad', output buffer memset failed.
